content/influxdb/clustered/install/set-up-cluster/test-cluster.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Test your InfluxDB Cluster
+description: >
+  Test to ensure your InfluxDB cluster can write and query data successfully.
+menu:
+  influxdb_clustered:
+    name: Test your cluster
+    parent: Set up your cluster
+weight: 250
+---
+
+With your InfluxDB cluster deployed and running, test to ensure you can
+successfully write and query data from InfluxDB.
+
+1. [Download and install influxctl](#download-and-install-influxctl)
+1. [Retrieve your cluster's admin token](#retrieve-your-clusters-admin-token)
+1. [Configure influxctl to connect to your cluster](#configure-influxctl-to-connect-to-your-cluster)
+1. [Create a new database](#create-a-new-database)
+1. [Write test data to the new database](#write-test-data-to-the-new-database)
+1. [Query the test data from your database](#query-the-test-data-from-your-database)
+
+## Download and install influxctl
+
+[`influxctl`](/influxdb/clustered/reference/cli/influxctl/) is a command line tool
+that lets you manage, write data to, and query data from your InfluxDB cluster
+from your local machine.
+
+<a href="/influxdb/clustered/reference/cli/influxctl/#download-and-install-influxctl" class="btn" target="_blank">Download and install influxctl</a>
+
+## Retrieve your cluster's admin token
```

Comment:
It would probably be a good idea to mention that whenever the customer sets up proper identity provider integration when securing their cluster they should revoke this admin token and regenerate it as per `influxdb/clustered/admin/bypass-identity-provider/#revoke-an-admin-token`